ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -111,3 +132,57 @@ Wait For Pods Numbers
   IF    '${status}' == 'False'
         Run Keyword And Continue On Failure    FAIL    Timeout- ${output} pods found with the label selector ${label_selector} in ${namespace} namespace
   END
+
+Apply DataScienceCluster CustomResource
+    [Documentation]
+    [Arguments]        ${dsc_name}=default
+    ${file_path} =    Set Variable    ods_ci/tasks/Resources/Files/
+    Log to Console    Requested Configuration:
+    FOR    ${cmp}    IN    @{COMPONENTS}
+        Log To Console    ${cmp} - ${COMPONENTS.${cmp}}
+    END
+    Create DataScienceCluster CustomResource Using Test Variables
+    ${yml} =    Get File    ${file_path}dsc_apply.yml
+    Log To Console    Applying DSC yaml
+    Log To Console    ${yml}
+    Run    oc apply -f ${file_path}dsc_apply.yml
```

Comment:
you could embed these lines into `Create DataScienceCluster CustomResource Using Test Variables`

Reply:
`Create` only creates the file, I don't think it should also apply it